vars/ploigosWorkflowExistingContainerImageScan.groovy

```diff
@@ -95,7 +95,12 @@ class WorkflowParams implements Serializable {
     /* Container image to use when creating a workflow worker
      * to run pipeline steps when no other specific container image has been
      * specified for that step. */
-    String workflowWorkerImageDefault = "ploigos/ploigos-ci-agent-jenkins:latest"
+    String workflowWorkerImageDefault = "ploigos/ploigos-base:latest"
+
+    /* Container image to use when creating a workflow worker
+     * to run pipeline steps when no other specific container image has been
+     * specified for that step. */
+    String workflowWorkerImageAgent = "ploigos/ploigos-ci-agent-jenkins:latest"
```

Comment:
You may also need to update the comment b/c it is a duplicate of the `workflowWorkerImageDefault`.